src/Item/ViewItem/ItemAcquisition/ItemAcquisition.js

```diff
@@ -40,9 +41,26 @@ const ItemAcquisition = ({ accordionId, itemId }) => {
     );
   }
 
-  const { orderLine, order, piece, vendor } = itemAcquisition;
+  const { orderLine, order, piece, vendor, finance, orderSetting } = itemAcquisition;
   const receiptDate = getDateWithTime(piece?.receivedDate);
 
+  const getAmountWithCurrency = (locale, currency, amount = 0) => (
+    Intl.NumberFormat(locale, { style: 'currency', currency }).format(amount)
```

Comment:
Please use `intl.formatNumber` to stay consistent with the rest of localization:
Either:
```
intl.formatNumber(amount, { style: 'currency', currency });
```
or
```
<FormattedNumber value={amount} style="currency" currency={currency} />
```